gdx/src/com/badlogic/gdx/maps/tiled/TmjMapLoader.java

```diff
@@ -0,0 +1,259 @@
+		if (source != null) {
+			FileHandle tsxFile = getRelativeFileHandle(tmjFile, source);
+			tileSet = json.parse(tsxFile);
+			if (tileSet.has("image")) {
+				String imageSource = tileSet.getString("image");
+				FileHandle image = getRelativeFileHandle(tsxFile, imageSource);
+				fileHandles.add(image);
+			} else {
+				for (JsonValue tile : tileSet.get("tile")) {
+					String imageSource = tile.getString("image");
+					FileHandle image = getRelativeFileHandle(tsxFile, imageSource);
+					fileHandles.add(image);
+				}
+			}
+		} else {
+			if (tileSet.has("image")) {
+				String imageSource = tileSet.getString("image");
+				FileHandle image = getRelativeFileHandle(tmjFile, imageSource);
+				fileHandles.add(image);
+			} else {
+				for (JsonValue tile : tileSet.get("tile")) {
+					String imageSource = tile.getString("image");
+					FileHandle image = getRelativeFileHandle(tmjFile, imageSource);
+					fileHandles.add(image);
+				}
+			}
+		}
+		return fileHandles;
+	}
+
+	@Override
+	protected void addStaticTiles (FileHandle tmjFile, ImageResolver imageResolver, TiledMapTileSet tileSet, JsonValue element,
+		JsonValue tiles, String name, int firstgid, int tilewidth, int tileheight, int spacing, int margin, String source,
+		int offsetX, int offsetY, String imageSource, int imageWidth, int imageHeight, FileHandle image) {
+
+		MapProperties props = tileSet.getProperties();
+		if (image != null) {
+			// One image for the whole tileSet
+			TextureRegion texture = imageResolver.getImage(image.path());
+
+			props.put("imagesource", imageSource);
+			props.put("imagewidth", imageWidth);
+			props.put("imageheight", imageHeight);
+			props.put("tilewidth", tilewidth);
+			props.put("tileheight", tileheight);
+			props.put("margin", margin);
+			props.put("spacing", spacing);
+
+			int stopWidth = texture.getRegionWidth() - tilewidth;
+			int stopHeight = texture.getRegionHeight() - tileheight;
+
+			int id = firstgid;
+
+			for (int y = margin; y <= stopHeight; y += tileheight + spacing) {
+				for (int x = margin; x <= stopWidth; x += tilewidth + spacing) {
+					TextureRegion tileRegion = new TextureRegion(texture, x, y, tilewidth, tileheight);
+					int tileId = id++;
+					addStaticTiledMapTile(tileSet, tileRegion, tileId, offsetX, offsetY);
+				}
+			}
+		} else {
+			// Every tile has its own image source
+			for (JsonValue tile : tiles) {
+				if (tile.has("image")) {
+					imageSource = tile.getString("image");
+
+					if (source != null) {
+						image = getRelativeFileHandle(getRelativeFileHandle(tmjFile, source), imageSource);
+					} else {
+						image = getRelativeFileHandle(tmjFile, imageSource);
+					}
+				}
+				TextureRegion texture = imageResolver.getImage(image.path());
+				int tileId = firstgid + tile.getInt("id");
+				addStaticTiledMapTile(tileSet, texture, tileId, offsetX, offsetY);
+			}
+		}
+	}
+
+	public static class Parameters extends BaseTmjMapLoader.Parameters {
+
```

Comment:
Maybe this is another candidate for a refactor, they all use the same parameters.
Wow... I am having the same issue. Locally I see
```
 public static class Parameters extends AssetLoaderParameters<TiledMap> {
		  /** generate mipmaps? **/
		  public boolean generateMipMaps = false;
		  /** The TextureFilter to use for minification **/
		  public TextureFilter textureMinFilter = TextureFilter.Nearest;
		  /** The TextureFilter to use for magnification **/
		  public TextureFilter textureMagFilter = TextureFilter.Nearest;
		  /** Whether to convert the objects' pixel position and size to the equivalent in tile space. **/
		  public boolean convertObjectToTileSpace = false;
		  /** Whether to flip all Y coordinates so that Y positive is up. All libGDX renderers require flipped Y coordinates, and thus
			* flipY set to true. This parameter is included for non-rendering related purposes of tmj files, or custom renderers. */
		  public boolean flipY = true;
	 }
```
But github is empty....